cmd/pod/getPod.go

```diff
@@ -30,6 +30,20 @@ var GetPodCmd = &cobra.Command{
 			}
 			row := []string{p.Id, p.Name, fmt.Sprintf("%d %s", p.GpuCount, p.Machine.GpuDisplayName), p.ImageName, p.DesiredStatus}
 			if AllFields {
+
+				var portEntries int = 0
+				if  p.Runtime != nil && p.Runtime.Ports != nil {
+					portEntries = len(p.Runtime.Ports)
+				}
+				ports := make([]string, portEntries)
+				for j:=0; j<portEntries; j++ {
+					var privpub string = "prv"
+					if p.Runtime.Ports[j].IsIpPublic {
+						privpub = "pub"
+					}
+					ports[j] = fmt.Sprintf("%s:%d->%d\u00A0(%s,%s)",  p.Runtime.Ports[j].Ip,  p.Runtime.Ports[j].PublicPort,  p.Runtime.Ports[j].PrivatePort,  privpub,  p.Runtime.Ports[j].PortType)
```

Comment:
I am deeply suspicious of `0xA0`, the non-breaking space.

Reply:
it's so it works with whatever table writer we're using, we can replace